### PR TITLE
docs: Contributing documentation

### PR DESCRIPTION
**Pull request by karencfv**
## Description
Adds contributing documentation.

## Related Issues
Closes: https://github.com/elastic/terraform-provider-ec/issues/59

## Types of Changes

- [x] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,18 +69,12 @@ resource "ec_deployment" "example_minimal" {
 
 ## Developer Requirements
 
-- [Terraform](https://www.terraform.io/downloads.html) 0.12+
+- [Terraform](https://www.terraform.io/downloads.html) 0.13+
 - [Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
```
Comment (marclop) [inline README.md:73]:
```suggestion
- [Go](https://golang.org/doc/install) 1.13+ (to build the provider plugin)
```

**Review by marclop — APPROVED** [review]
Looks great!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  - [Commit Messages](#commit-messages)
+- [Setting up a dev environment](#setting-up-a-dev-environment)
+  - [Environment prerequisites](#environment-prerequisites)
+- [Development](#development)
+  - [Running tests](#running-tests)
+  - [Build terraform-provider-ec locally with your changes](#build-terraform-provider-ec-locally-with-your-changes)
+
+## Reporting Issues
+
+If you have found an issue or defect in `ecctl` or the latest documentation, use the GitHub [issue tracker](https://github.com/elastic/terraform-provider-ec/issues) to report the problem. Make sure to follow the template provided for you to provide all the useful details possible.
```
[the remark this answers is not in the document]
Comment (karencfv) [reply CONTRIBUTING.md:17]: Oops 😄

**Review by marclop — APPROVED** [review]
🚀 📜

**Pull request merged**